src/main/java/com/shrek/task00003/Uniqueness.java

```diff
@@ -0,0 +1,79 @@
+package com.shrek.task00003;
+
+import javax.annotation.Nonnull;
+
+import java.util.*;
+
+/**
+ * Класс в конструкторе принимает список строк. Необхдимо реализовать 3 метода:
+ * 1. Есть ли в списке дубликаты
+ * 2. Сколько дублирующихся элементов
+ * 3. Сколько каждый элемент раз встречается
+ * <p>
+ * Написать тесты
+ */
+public class Uniqueness {
+    public List<String> elements;
+    public Set<String> elementsInSet;
+    public List<String> uniquessElements;
+
+    public Uniqueness(@Nonnull List<String> elements) {
+        this.elements = elements;
+        this.elementsInSet = new TreeSet<String>(elements);
+        this.uniquessElements = new ArrayList<String>(elementsInSet);
+    }
+
+    public int duplicateCounter() {
+        int duplicateCount = 0;
+        for (int i = 0; i < elementsInSet.size(); i++) {
+            int count = 0;
+            for (int j = 0; j < elements.size(); j++) {
+                if (elements.get(j).equals(uniquessElements.get(i))) {
+                    count++;
+                }
+            }
+            if (count > 1) {
+                duplicateCount++;
+            }
+        }
+        return duplicateCount;
+    }
+
+    public boolean duplicateCheck() {
+        if (elements.size() < 2) {
+            return false;
+        }
+        for (int i = 0; i < elements.size(); i++) {
+            for (int j = 0; j < elements.size(); j++) {
```

Comment:
We should check all pairs. So if we checked i=0 and j=1 than we shouldn't check i=1 and j=0 because this is the same.
Think how to do it